### PR TITLE
Submissions live feed

### LINKED ISSUE #8: Submissions live feed per guide
**AC**
- It should include a global progress guide.
- The feed should update when a new submission occurs.
- To quickly determine which students need help, they should be ordered by passed exercises count.

### PR DESCRIPTION
**Pull request by faloi**
Closes #8 

### Compact version (default)
image

### With details
image

### Sorted by students' name
image

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -78,7 +79,12 @@
           <script src="app/guides/guides.controller.js"></script>
           <script src="app/guides/guides.js"></script>
           <script src="app/guides/guides.service.js"></script>
+          <script src="app/live/live.controller.js"></script>
+          <script src="app/live/live.js"></script>
           <script src="app/main/main.js"></script>
+          <script src="app/models/guide.model.js"></script>
+          <script src="app/models/student.model.js"></script>
+          <script src="app/models/students.service.js"></script>
```
Comment (flbulgarelli) [inline client/index.html]: Isn't any way of not adding all the js manually?

File: client/components/navbar/navbar.controller.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ angular.module('mumukiTeacherToolsApp')
       'title': 'Agregar alumnos',
       'link': '/courses/new'
     }, {
+      'title': 'Soluciones en vivo',
+      'link': '/live'
```
Comment (flbulgarelli) [inline client/components/navbar/navbar.controller.js]: Just curious: do we have i18n here?

File: client/index.html (hunk 1 above)
Comment (faloi) [inline client/index.html]: It's not manual, _grunt_ adds them automatically between the `<!-- build:js(.tmp) app/app.js -->` tags.

File: client/app/live/live.controller.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    $scope.sortingCriteria = (student) =>
+      $scope.sortType === 'name' ? student.fullName() : student.pointsFor($scope.selectedGuide.exercises);
+
+    $scope.statusIconFor = (status) => {
+      switch (status) {
+        case 'failed':
+        case 'errored':
+          return 'fa-times';
+
+        case 'passed':
```
Comment (flbulgarelli) [inline client/app/live/live.controller.js]: Don't we already have this switch elsewhere?

**Comment by flbulgarelli** [comment]
Looks awesome. How about giving a different backound color to cards instead of using different font colors?

File: client/components/navbar/navbar.controller.js (hunk 2 above)
Comment (faloi) [inline client/components/navbar/navbar.controller.js]: Not yet :cry:

**Comment by flbulgarelli** [comment]
Also, we should use yellow instead of black for those who have intermidiate results.

**Comment by faloi** [comment]
> Looks awesome. How about giving a different backound color to cards instead of using different font colors?

I tried it, but doesn't look very well.

image

> Also, we should use yellow instead of black for those who have intermidiate results.

Also tried it, but I think black looks better (screenshot coming).

**Comment by flbulgarelli** [comment]
Hahah, I dont't mean those pastel colors. I Mean light colors.

**Comment by PalumboN** [comment]
This is awesome. You rock :guitar:

**Comment by aguspina** [comment]
I really love you

**Comment by flbulgarelli** [comment]
OK, let's merge this. We will see if we need any color change later

**Pull request merged**